minio-kes/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:latest
+RUN apk add --no-cache bash
+
+WORKDIR /kes
+
+RUN wget https://github.com/minio/kes/releases/download/v0.17.6/kes-linux-amd64 && \
+    mv kes-linux-amd64 kes && \
+    chmod +x kes && \
+    ./kes --version
+
+EXPOSE 7373
+
+COPY kes-server-config.yaml init.sh ./
+
+ENTRYPOINT ["./init.sh"]
```

Comment:
there is an official docker image, please use it - https://github.com/minio/kes/blob/master/README.md#docker

Reply:
please add a minio-kes/README.md explaining what is this image, but ideally we should just use standard kes image and not have to implement so much code for it

Reply:
The official one doesn't run a `kes server`, nor does it accept any of the required env vars.
See: https://github.com/minio/kes/blob/master/Dockerfile | https://github.com/minio/kes/blob/master/Dockerfile.release
For our use case, we need a running `kes server` to make `minio` able to communicate with it for en/decryption.

---

Actually, the code is at one place for testing purposes only.
Some of it should be at the `minio` side.

Reply:
it contains the kes binary, that's all you need, you can add command arguments and environment vars without having to build a new image

if you need an init script you can build a docker image based on the official one with FROM and just add the init script

regarding the init script - please describe what it does, does it need to run each time the image starts? is it some one-off thing? I think it would be better at this stage to not implement too much code, and just write a documentation about how to run it. Later we can see what we want to implement to make it run at scale.

Reply:
Right. Makes sense. Thanks! 
Sure, I'll update it accordingly. :+1: